next-env.d.ts

```diff
@@ -1,5 +1,4 @@
 /// <reference types="next" />
-/// <reference types="next/types/global" />
```

Comment:
Waarvoor is deze regel verwijderd? Was dit door Next gedaan?

Reply:
Yes. I didn't change this myself, so assumed is was due to the update.